tests/baselines/reference/privateNameDuplicateField.js

```diff
@@ -557,7 +551,7 @@ function Method() {
     _d = A_Method_StaticSetter, _A_Method_StaticSetter_instances = new WeakSet();
 }
 function Getter() {
-    var _A_Getter_Field_instances, _A_Getter_Field_foo_get, _A_Getter_Field_foo, _A_Getter_Method_instances, _A_Getter_Method_foo_get, _A_Getter_Method_foo, _A_Getter_Getter_instances, _A_Getter_Getter_foo_get, _A_Getter_Getter_foo_get_1, _A_Getter_Setter_instances, _A_Getter_Setter_foo_get, _A_Getter_Setter_foo_set, _A_Getter_StaticField_instances, _a, _A_Getter_StaticField_foo_get, _A_Getter_StaticField_foo, _A_Getter_StaticMethod_instances, _b, _A_Getter_StaticMethod_foo_get, _A_Getter_StaticMethod_foo, _A_Getter_StaticGetter_instances, _c, _A_Getter_StaticGetter_foo_get, _A_Getter_StaticGetter_foo_get_1, _A_Getter_StaticSetter_instances, _d, _A_Getter_StaticSetter_foo_get, _A_Getter_StaticSetter_foo_set;

```

Comment:
is this rearrangement of foo_get_1 to foo_1_get correct?

Reply:
Yes. This is an intended consequence of the changes I made to allow prefixes and suffixes in name generation.

Reply:
However, it does look like its unused in any case.